src/Aspire.Hosting.Azure/Provisioning/Provisioners/AzureProvisioner.cs

```diff
@@ -256,13 +256,14 @@ private async Task ProcessResourceAsync(IConfiguration configuration, Lazy<Task<
 
     private async Task<ProvisioningContext> GetProvisioningContextAsync(JsonObject userSecrets, CancellationToken cancellationToken)
     {
-        var credential = new DefaultAzureCredential(new DefaultAzureCredentialOptions()
-        {
-            ExcludeManagedIdentityCredential = true,
-            ExcludeWorkloadIdentityCredential = true,
-            ExcludeAzurePowerShellCredential = true,
-            CredentialProcessTimeout = TimeSpan.FromSeconds(15)
-        });
+        // var credential = new DefaultAzureCredential(new DefaultAzureCredentialOptions()
+        // {
+        //     ExcludeManagedIdentityCredential = true,
+        //     ExcludeWorkloadIdentityCredential = true,
+        //     ExcludeAzurePowerShellCredential = true,
+        //     CredentialProcessTimeout = TimeSpan.FromSeconds(15)
+        // });
+        var credential = new AzureCliCredential();
```

Comment:
???

Reply:
my personal azure subscription where I test things is not related to the ambient login/visualstudio login I have on my work laptop, which is linked to a different tenant. Using the DefaultAzureCredential means it hits my work subscription first. I spoke to @davidfowl about this -- there needs to be a way to use a credential other than the default for deployment. This is a temporary hack :)

Reply:
![image](https://github.com/dotnet/aspire/assets/1844001/ab4f3eb3-d522-47ae-b56d-c6f21278d598)